examples/publish-ci/cra4/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@reduxjs/toolkit": "^2.0.0-beta.2",
+    "@reduxjs/toolkit": ">=2.0.0-beta.3 || >=2.0.0-rc.0 || ^2.0.0",
```

Comment:
Why is this change necessary if the contribution is to update the Yarn version?
It's just a doubt.